build.gradle

```diff
@@ -63,13 +66,18 @@ tasks {
     }
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
 repositories {
     jcenter()
 }
 
 dependencies {
-    compile 'net.i2p.crypto:eddsa:0.3.0'
-    testImplementation 'junit:junit:4.12'
+    implementation 'net.i2p.crypto:eddsa:0.3.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.7.0'
```

Comment:
The bulk of the diff for this pull request is syntax changes needed to update from junit4 to junit5.